cmd/manta-init.js

```diff
@@ -636,8 +636,31 @@ var pipelineFuncs = [
 	},
 
 	function checkMantaApplicationVersion(_, cb) {
```

Comment:
nit: I'd consider changing this function name to "ensureMantavMetadatum" or something to hint that it can *set* a value now.

Reply:
Good idea, I'll change to exactly that.